### ISSUE TITLE
<x-splade-content may break/interfere with app styling

### ISSUE THREAD

**Issue by J87NL** [protonemedia/laravel-splade#416]
- Laravel Version: 10.7.1
- PHP Version: 8.2.5
- Splade JS Version (npm): 1.4.7
- Splade PHP Version (composer): 1.4.7
- Dev environment: Ubuntu 22.04 with Apache2

### Description:
We load and show the content of emails using `x-splade-content` like:

```blade
    <x-splade-content class="prose max-w-none" :html="$item->body" />
```

Sometimes the styling inside this message breaks the styling of our app, like:

![afbeelding](https://user-images.githubusercontent.com/16107428/236860126-1f0e15b3-c79f-428f-9367-aa32c90abe1a.png)   ==>    ![afbeelding](https://user-images.githubusercontent.com/16107428/236860197-6a5b1e99-744a-48fa-87ab-b05619ed153a.png)

Is there something we can do about this?

**Comment by J87NL** [comment]
Hi Pascal, thanks for your reply!

Basically any content that contains anything like:
```html
<style>
    a {
        color: red !important;
    }
</style>
```
For obvious reasons this gets loaded into the DOM and is interfering with the other HTML. I added the `!important` in the example above for demo purposes.

The possibility of having style and script tags in content that can be added by end users is a bad thing of course, we are aware of that.
We've added a bit of cleanup to prevent this issue, the line below is part of this and solves this issue for us:
```php
    preg_replace('/(<(script|style)\b[^>]*>).*?(<\/\2>)/is', '$1$3', $body)
```

At this point this issue feels like expected (or even desired maybe?) behavior and I think it can be closed.
